### PR TITLE
Relax the requirement that all Standing Committees Have an SSC Representative

### PR DESCRIPTION
**Pull request by blink1073**
### Background or context to help others understand the change.
Not all Standing Committees will require an SSC representative.

### A brief summary of the change.
Align the language for Standing Committee SSC representation with the Working Groups

> ## The process ❗
> 
> The process for changing the governance pages is as follows:
> 
> * Open a pull request **in draft state**. This triggers a discussion and iteration phase
>   for your proposed changes.
> * When you believe enough discussion has happened,
>   **move the pull request to an active state**. This triggers a vote.
> * During the voting phase, no substantive changes may be made to the pull request.
> * The Executive Council and Software Steering Council will vote, and at the end of voting the pull request is merged or closed.
> 
> The discussion phase is meant to gather input and multiple perspectives from the community.
> Make sure that the community has had an opportunity to weigh in on
> the change **before calling a vote**. A good rule of thumb is to ask several Council
> members if they believe that it is time for a vote, and to let at least one person review
> the pull request for structural quality and typos.

## Votes

> The decision will be determined by a simple majority of non-blank votes for binary decisions (i.e., approving a proposal) and ranked choice for multi-class decisions (one among many, or several among many). The sponsor may update the proposal at any point during the voting period, in which case the voting period will be reset.

**The voting period for the PR is from 7 Mar 2023 to 14 Mar 2023 anywhere on 🌍**

@afshin 
- [ ] Yes
- [ ] No
- [ ] Abstain

@Ruv7
- [ ] Yes
- [ ] No
- [ ] Abstain

[@ellisonbg](https://github.com/ellisonbg)
- [x] Yes
- [ ] No
- [ ] Abstain

[@fperez](https://github.com/fperez)
- [x] Yes
- [ ] No
- [ ] Abstain

[@jasongrout](https://github.com/jasongrout)
- [ ] Yes
- [ ] No
- [ ] Abstain

[@blink1073](https://github.com/blink1073)
- [ ] Yes
- [ ] No
- [ ] Abstain

[@isabela-pf](https://github.com/isabela-pf)
- [ ] Yes
- [x] No
- [ ] Abstain

[@ivanov](https://github.com/ivanov)
- [ ] Yes
- [ ] No
- [ ] Abstain

[@johanmabille](https://github.com/johanmabille)
- [ ] Yes
- [ ] No
- [ ] Abstain

[@echarles](https://github.com/echarles)
- [ ] Yes
- [ ] No
- [ ] Abstain

[@rpwagner](https://github.com/rpwagner)
- [ ] Yes
- [ ] No
- [ ] Abstain

[@zsailer](https://github.com/zsailer)
- [ ] Yes
- [ ] No
- [ ] Abstain

[@willingc](https://github.com/willingc)
- [ ] Yes
- [x] No
- [ ] Abstain

[@ibdafna](https://github.com/ibdafna)
- [x] Yes
- [ ] No
- [ ] Abstain

[@minrk](https://github.com/minrk)
- [ ] Yes
- [ ] No
- [ ] Abstain

[@fcollonval](https://github.com/fcollonval)
- [ ] Yes
- [ ] No
- [ ] Abstain

[@SylvainCorlay](https://github.com/SylvainCorlay)
- [x] Yes
- [ ] No
- [ ] Abstain

### CODE REVIEW

File: standing_committees_and_working_groups.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Standing Committees have the following unique properties:
 
 - Standing Committees focus on areas critical to the governance of the project and thus are established and dissolved by joint decisions of the SSC and EB.
 - Changes to the charters of Standing Committees are approved by joint decisions of the SSC and EB.
-- Standing Committees always have a representative on the SSC, elected by the Standing Committee Council.
+- Optionally, the EB may nominate certain Standing Committees to have a representative on the SSC. Once approved by the SSC to have a representative on the SSC, Standing Committees elect and maintain their representative to the SSC using the Decision-Making Guide.
```
Comment (jasongrout) [inline standing_committees_and_working_groups.md:28]:
```suggestion
- Optionally, the EB may nominate certain Standing Committees to have a representative on the SSC. Once approved by the SSC to have a representative on the SSC, Standing Committee Councils elect and maintain their representative to the SSC using the Decision-Making Guide.
```

**Comment by blink1073** [comment]
@jupyter/executive-council @jupyter/software-steering-council

**Comment by westurner** [comment]
From 
https://github.com/damianavila/RISE/issues/635#issuecomment-1447977882 :

> > * https://github.com/jupyter
> >   * https://github.com/jupyter/governance
> 
> https://jupyter.org/governance/overview.html#other-major-components-of-the-organization :
> 
> > * Distinguished Contributors
> > * Standing Committees and Working Groups
> > * Software Subprojects
> 
> https://jupyter.org/governance/software_subprojects.html#responsibilities-of-jupyter-subprojects
> 
> https://jupyter.org/governance/list_of_subprojects.html#list-of-official-jupyter-subprojects :
> 
> > * Official Subprojects with SSC representation
> > * *Official Subprojects without SSC representation*

**Comment by isabela-pf** [comment]
This is possibly a naive question, but since I haven't heard anything about this prior I'd like to know more about the motivation. In reading the PR I'm curious about the following.
- Why is there interest in not having all Standing Committees be represented on the Steering Council? 
- If the EC decides which groups may or may not have a representative, would there be any case where a group may want a representative and not get one? Is there some kind of implicit criteria?

Thanks in advance for the clarification! 🌻

**Comment by rpwagner** [comment]
My question is somewhat similar to @isabela-pf's: would SSC representation be based on whether or not a standing committee should have voting rights in governance decisions? This may help clarify the difference between advisory (say Working Groups) or decision making (say Standing Committees) groups.

**Comment by jasongrout** [comment]
Thank you for bringing up this question, and actually that reminds us that the process listed above is to open in draft mode for discussion, and then call a vote once the discussion has happened.

@blink1073 - I'll switch this to draft mode to have the discussion phase first.

**Comment by jasongrout** [comment]
Since there is discussion that should happen, I propose that we take a step back, have the discussion, and then reset the voting period.

**Comment by willingc** [comment]
Why are we relaxing the requirement for a Standing Committee to have representation? I'm very concerned that this change will minimize the voices of people working on non-software development aspects of the project. We know how important things beyond software development are to the health and success of an open source project. 

If an area is important enough to have a Standing Committee, the area should have representation. This is distinctly different than a working group which will likely have a shorter lifespan.

File: standing_committees_and_working_groups.md [hunk 2]
```diff
@@ -18,21 +18,20 @@ Standing Committees and Working Groups both:
 - Have charters that define their scope and purpose. Within those charters, they are free to make decisions autonomously.
 - Carry out their functions in a manner that is as open and transparent as possible given the unique dimensions of their charter.
 - Maintain a public GitHub repository with basic information (council members, charter, public meetings, SSC representative, etc.) for purposes of transparency and consistency.
+- Optionally, the EC may nominate certain Working Groups or Standing Committee councils to have a representative on the SSC. Once approved by the SSC to have a representative on the SSC, the Working Group or Standing Committee council will elect and maintain their representative to the SSC using the Decision-Making Guide.
```
Comment (willingc) [inline standing_committees_and_working_groups.md:21]:
```suggestion
- A Standing Committee which is created by a joint vote of the EC and SSC will have a representative serving on the SSC. If approved by the EC and SSC that a Standing Committee should be formed, the Standing Committee will have a representative on the SSC. The Standing Committee will elect and maintain their representative to the SSC using the Decision-Making Guide.
- Optionally, the EC may nominate certain Working Groups to have a representative on the SSC.  If approved by the SSC to have that a Working Group should have a representative on the SSC, the Working Group will elect and maintain their representative to the SSC using the Decision-Making Guide.
```

**Comment by blink1073** [comment]
Thanks for the feedback everyone, I think the original idea here was that the Standing Committees shouldn't necessarily be charged with things like reviewing JEPs, but I agree that they should be involved in governance.  I'll take it as an action item to discuss at the joint EC/SSC meeting tomorrow.

**Comment by willingc** [comment]
@blink1073 As far as JEPs go, Standing Committee representatives should have the ability to review JEPs since any individual can review a JEP for understanding and clarity. In the Python world, we added a section on how to teach to every PEP so that changes would be more accessible to members of the Python community.

**Comment by ellisonbg** [comment]
Great questions! I can give a bit more background about where this PR came from...

* There were a few weeks where different members of the governance working group were out. Across those weeks, we ended up merging a few things into the governance documents were not fully self consistent and we didn't notice until later.
* One of those is that the [Community Advisory Panel](https://jupyter.org/governance/list_of_standing_committees_and_working_groups.html#community-advisory-panel) was listed as a Standing Committee (see the list [here](https://jupyter.org/governance/list_of_standing_committees_and_working_groups.html#)). I believe the reason we did that is to capture the idea that it is a non-optional and long term thing we want to have (as opposed to the more optional and transient Working Groups). However, by labeling it as a Standing Committee we forced it to have a couple of things that aren't compatible with the traditional meaning of an advisory group: 1) it has to have a council for formal decision making, 2) that council has formal decision making responsibilities in the project, 3) it has a representative on the SSC, 3) that representative has a formal vote in a [number of matters](https://jupyter.org/governance/software_steering_council.html). Typically an advisory group involved a relatively small time commitment and doesn't have formal decision making responsibilities. As such, we plan on removing the Advisory Panel from the list of Standing Committees and put it in a separate part of the governance documents that keeps it as a permanent part of the governance model, but clarifies its role as being advisory in nature, with a small time commitment and no formal decision making responsibilities.
* That discussion caused us to go back and look at the list of Standing Committees to make sure that it made sense for all of them to always have an SSC representative (the other Standing Committees are: DEI, CoC Reponse, and Conflict of Interest). Our sense was that some of these groups would be small, focused, and have a relatively small time commitment. Given that the [responsibilities of the SSC](https://jupyter.org/governance/software_steering_council.html) are significant, our sense was that it was too inflexible to require all of them to have an SSC representative.
* The original vision remains the same: we (summarizing the discussions of the governance working group and EC) believe that it is critical for non-software stakeholders to have visibility into and a formal vote in software matters. The mechanism remains the same: that Standing Committees and Working Groups can, at the discretion of EC+SSC, have an SSC representative.
* While we haven't written it down, I think we imagine that as the Standing Committees and Working Group are writing their charters and establishing their initial councils that the EC/SSC would discuss with those initial members if it make sense for them to have an SSC representative. 

It may make sense for us to make changes to this PR to reflect these different dimensions as they all work together in the overall picture.

File: standing_committees_and_working_groups.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 In addition to the software work on Jupyter that is coordinated through the [Software Steering Council](software_steering_council.md) (SSC), much of the project’s work expands beyond software. Examples include code of conduct incident response, diversity and inclusion, operations, legal, fundraising, events, community, and marketing. Standing Committees and Working Groups carry out this non-software related work of the project by delegation from the [Executive Council](executive_council).
 
-The primary difference between Standing Committees and Working Groups is that Standing Committees are intended to be permanent; they are only created and dissolved by a joint vote of the EB and SSC. In contrast, Working Groups can be created and dissolved by the EB acting alone.
+The primary difference between Standing Committees and Working Groups is that Standing Committees are intended to be permanent; they are only created and dissolved by a joint vote of the EC and SSC. In contrast, Working Groups can be created and dissolved by the EC acting alone.
```
[the remark this answers is not in the document]
Comment (ellisonbg) [reply standing_committees_and_working_groups.md:7]: Thanks, we keep finding outdated acronyms like this :-)

**Comment by fperez** [comment]
Thanks @ellisonbg for the detailed clarification! To echo/expand a bit, the intention was never to remove representation/voices, much less to do so for non-software work. We are trying to continue moving the project more and more towards both recognizing and better supporting non-software work (I say that as someone who hasn't had much code contributions in a long time 🙂). In a project and community of our size and scope, while software is critical, it's also only a small part of the whole picture.

I suggest we iterate on language that addresses the practical considerations we had in mind while making it absolutely clear that there's no removal of voice/governance for any stakeholder in the project. Sorry for not being more careful in the wording up front and causing concern, and thanks everyone for the feedback!

If anything, this shows the value of having these key governance changes go through public review with the combined set of eyes of the community - regardless of the authors' intents, we want to make sure that what ends up committed meets the needs of all our stakeholders. Thanks a lot @isabela-pf @rpwagner @willingc for pitching in to review!!

**Comment by isabela-pf** [comment]
Thank you all for the taking the time to explain the motivation and background! I'm hearing a few different factors are involved, but it sounds like the main concern here is to respect people's time and not create perceived red tape where it clearly isn't necessary or standard. While this may not address every concern, it sounds like this might be best resolved by adjusting governance so that
- Standing Committees and Working Groups decide on their own whether or not they want to exercise their right to have a representative on the SSC. By "on their own" I mean they are the only ones who get to make the decision; they may want to consult other governance groups for more context. I don't think this should be the EC's decision. 
- Standing Committees and Working Groups can decide to edit their charter and reinstate their right to have a SSC representative at any time for any reason.

I think this is similar what @ellisonbg listed above, just with extra notes.

> While we haven't written it down, I think we imagine that as the Standing Committees and Working Group are writing their charters and establishing their initial councils that the EC/SSC would discuss with those initial members if it make sense for them to have an SSC representative.

This approach would still do what the PR title says—relax the requirement that all standing committees are required to have a SSC representative—while ensuring the decision of representation remains with the people that would be represented. That's my primary concern with the proposed changes as they are now.

**Comment by blink1073** [comment]
That is a very good idea @isabela-pf!  Unfortunately we ran out of time to discuss it this week at the Executive Council meeting, I'll carry the action item to next week.

**Comment by fperez** [comment]
As a quick update on this, and to reiterate that the intent of this PR was _never_ to remove any representation from the DEI committee in the SSC - @marthacryan is the DEI Standing Committee representative at the SSC, as per all original plans. @marthacryan - you should have been added to the relevant mailing lists/etc, but please let us know if there's anything missing.

**Review by Zsailer — APPROVED** [review]

**Comment by blink1073** [comment]
Closing since I'm no longer involved with governance and there was no clear consensus on this change.

**Pull request closed without merging**